src/transformers/image_transforms.py

```diff
@@ -420,3 +426,106 @@ def center_crop(
         new_image = to_pil_image(new_image)
 
     return new_image
+
+
+def _center_to_corners_format_torch(bboxes_center: "torch.Tensor") -> "torch.Tensor":
+    x_c, y_c, w, h = bboxes_center.unbind(-1)
+    b = [(x_c - 0.5 * w), (y_c - 0.5 * h), (x_c + 0.5 * w), (y_c + 0.5 * h)]
+    return torch.stack(b, dim=-1)
+
+
+def _center_to_corners_format_numpy(bboxes_center: np.ndarray) -> np.ndarray:
+    x_c, y_c, w, h = bboxes_center.T
+    bboxes_corners = np.stack([x_c - 0.5 * w, y_c - 0.5 * h, x_c + 0.5 * w, y_c + 0.5 * h], axis=-1)
+    return bboxes_corners
+
+
+def _center_to_corners_format_tf(bboxes_center: "tf.Tensor") -> "tf.Tensor":
+    x_c, y_c, w, h = tf.unstack(bboxes_center, axis=-1)
+    bboxes_corners = tf.stack([x_c - 0.5 * w, y_c - 0.5 * h, x_c + 0.5 * w, y_c + 0.5 * h], axis=-1)
+    return bboxes_corners
+
+
+# 2 functions below inspired by https://github.com/facebookresearch/detr/blob/master/util/box_ops.py
+def center_to_corners_format(bboxes_center: TensorType) -> TensorType:
+    """
+    Converts bounding boxes from center format (center_x, center_y, width, height) to corners format (x_0, y_1, x_1,
+    y_1).
```

Comment:
```suggestion
    Converts bounding boxes from center format (center_x, center_y, width, height) to corners format `(left, top, right, bottom)`.
```
This is clearer to me than x_0/y_1